packages/fast-form-generator-react/src/form/form-item.align-horizontal.style.ts

```diff
@@ -1,6 +1,13 @@
-import { applyInputBackplateStyle, applyLabelStyle, applyWrapperStyle, colors } from "../utilities/form-input.style";
+import {
+    applyInputBackplateStyle,
+    applyInputWrapperStyle,
+    applyLabelStyle,
+    applyWrapperStyle,
+    colors
+} from "../utilities/form-input.style";
 import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
 import { IFormItemAlignHorizontalClassNameContract } from "../class-name-contracts/";
+import { toPx } from "@microsoft/fast-jss-utilities";
```

Comment:
do you actually need `toPx`?

Reply:
no we don't I added for some inline styles that I ended up moving.